### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by antonjb**
Updates all the dependencies used in the project. Including Next and
opts into using Next SWC for building.

Removes dependencies that were no longer used or required with Next
updates, such as use-script which is replace with Next Script.

Several scripts are moved into new locations as required by Next. See
the changes in both _app and _document.

The Next update required a new Jest configuration.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
-  "presets": ["next/babel"]
```
Comment (antonjb) [inline .babelrc:2]: Removed to enable Next SWC

File: components/global/meta.tsx [hunk 2]
```diff
@@ -68,24 +68,6 @@ export const Meta: React.FC<MetaArgs> = ({
       <link rel="preconnect" href="https://www.google-analytics.com"></link>
       <link rel="preconnect" href="https://fonts.gstatic.com"></link>
       <link rel="preconnect" href="https://az416426.vo.msecnd.net"></link>
-
-      <link href="https://fonts.googleapis.com/css?family=Hind:400,500,700&display=swap" rel="stylesheet" />
```
Comment (antonjb) [inline components/global/meta.tsx:72]: Moved to the _document as required by Next.

File: components/global/meta.tsx [hunk 3]
```diff
@@ -68,24 +68,6 @@ export const Meta: React.FC<MetaArgs> = ({
...
-
-      {appConfig.instrumentationKey && (
```
Comment (antonjb) [inline components/global/meta.tsx:74]: Moved to a Next Script component in the _app

File: pages/_app.tsx [hunk 4]
```diff
@@ -1,19 +1,43 @@
...
+  emotionCache.compat = true
+
   return (
     <ConfigProvider>
-      <ThemeProvider theme={theme}>
-        <Global styles={CSSReset} />
-        <Global styles={globalCSS} />
-        <Component {...pageProps} />
-      </ThemeProvider>
+      <CacheProvider value={emotionCache}>
```
Comment (antonjb) [inline pages/_app.tsx:19]: This is to remove Emotion's, incorrect, :first-child warnings.

**Review by robdmoore — APPROVED** [review]

**Pull request merged**